### PR TITLE
updated description of GBD model for diarrheal diseases

### PR DESCRIPTION
**Pull request by beatrixkh**
I might have had this in a previous PR, but it got lost when I moved diarrheal diseases into a sub folder. If so, I'm sorry for the redundancy!

### CODE REVIEW

File: docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst [hunk 1]
```diff
@@ -19,37 +19,27 @@ or more loose stools in a 24-hour period." (p. 88 of
 Modeling Diarrheal Diseases in GBD 2017
 ---------------------------------------
 
+The GBD diarrheal diseases model follows a standard GBD framework, including a cause of death (CoD) model and
+a nonfatal model.
 
+The CoD model estimates the cause-specific mortality rate (CSMR) within the total population, and a cause fraction.
+These estimates are based on vital registration and autopsy data.
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst]: and _verbal_ autopsy data?

File: docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst [hunk 2]
```diff
@@ -19,37 +19,27 @@ or more loose stools in a 24-hour period." (p. 88 of
...
-Regarding the duration of a bout of diarrhea, "the mean duration was
-the duration in days, an average of 4.3 (4.2 4.4)". (p. 89, based on a
-paper referenced there).
-For GBD 2017, the remission period was modeled as 5 days.
-Since this assumption gets into the DisMod
-model, we will use the remission rate that comes from DisMod.
+The nonfatal model is run in DisMod. The primary inputs are prevalence epi data, for which self-reported prevalence is
+the reference definition, and the CSMR estimates from CodCorrect. Separately,
+the ratio of mild/moderate/severe diarrhea is estimated, based on data from a
+systematic review. These estimates do not vary by age/sex/location/year, and are applied to the prevalence
```
Comment (aflaxman) [inline docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst]: These _severity ratio estimates_ do not vary by ... --- to make it clear that you are not also talking about the DisMod or CodCorrect models.

Comment (collijk) [reply docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst]: It's a surpise to me that they are constant across demography.  Did you confirm this in the data? I thought they were based on models of healthcare utilization in the country.  @ctroeger your expertise is definitely appreciated here.

File: docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst (hunk 1 above)
Comment (beatrixkh) [reply docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst]: Yes! Verbal autopsy; thanks for the catch

File: docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst (hunk 2 above)
Comment (beatrixkh) [reply docs/source/models/gbd2017/causes/diarrheal_diseases/index.rst]: @ctroeger said that the parent model has a flat estimate for severities. those flat estimates _are_ based on a systematic review which relies in part on hospitalization. The etiologies are far more complicated--and that was where, for their CoD model, they assumed that severe and fatal cases=hospitalized cases @collijk

**Pull request merged**